### ISSUE TITLE
BUG: doesn't find directories

### ISSUE THREAD

**Issue by Tarick** [r3inbowari/go-mod-explorer#1]
When I try to expand the package, it shows

```
ENOENT: no such file or directory, scandir 'file:///home/tarick/go/pkg/mod/gopkg.in/yaml.v2@v2.4.0'
```

Needless to say that the dir does exist.

**Comment by r3inbowari** [comment]
Thank you for your feedback. What system are you running on？

**Comment by r3inbowari** [comment]
Whether all the dependencies can not to load?

**Comment by Tarick** [comment]
Linux (Ubuntu 21.10), the latest VSCode. All go.mod dependencies are valid and the build passes, so code is ok. Yaml package is just the example, all other tries to expand other modules also return such messages.
If you meant the extension dependencies, then as I see it doesn't declare any dependencies in .vsixmanifest.

**Comment by r3inbowari** [comment]
hi.
i meant can you see any dependencies loaded(displayed) on mod explorer.

**Comment by r3inbowari** [comment]
sorry, i never try it on linux, plz wait a moment

**Comment by Tarick** [comment]
I see the dependencies loaded, but when I click on the little ">" sign to expand it, the mentioned error is shown. If this wasn't intended to work in this first version, then probably it shouldn't be shown.

**Comment by r3inbowari** [comment]
I have tried to fix this problem, you can try the new v0.1.1.

**Comment by Tarick** [comment]
Yes, everything works correctly now, thanks!
